variables.tf

```
@@ -30,4 +30,8 @@ variable "published" {
   description = "If the product should be published"
 }
 
-
+variable "product_access_control_groups" {
+  type        = list(string)
+  default     = [""]
```

shouldn't this be an empty list? so that it does nothing by default?

```suggestion
  default     = []
```